src/ReceivingList/ReceivingListContainer.js

```diff
@@ -85,7 +90,9 @@ const ReceivingListContainer = ({ mutator, location }) => {
           ...orderLinesResponse.reduce((acc, orderLine) => {
             acc[orderLine.id] = {
               ...orderLine,
-              locations: orderLine.locations.map(({ locationId }) => newLocationsMap[locationId].name),
+              locations: orderLine.locations.map(
+                ({ locationId }) => newLocationsMap[locationId]?.name || invalidReferenceMessage,
```

Comment:
```suggestion
                ({ locationId }) => newLocationsMap[locationId]?.name ?? invalidReferenceMessage,
```
Could there be an empty string in `name`

Reply:
location with empty name? nope, it's required

Reply:
but ?? makes sense